src/utils/config/__tests__/example/v004.ts

```diff
@@ -17,6 +17,9 @@ export const configExample: Config = {
     openrouter: {
       apiKey: undefined,
     },
+    ollama: {
```

Comment:
You shouldn't change either `v004.ts` or `v005.ts`. because your config schema version should be 6